rust/kcl-lib/src/std/extrude.rs

```diff
@@ -31,7 +33,15 @@ use crate::{
 
 /// Extrudes by a given amount.
 pub async fn extrude(exec_state: &mut ExecState, args: Args) -> Result<KclValue, KclError> {
-    let sketches = args.get_unlabeled_kw_arg("sketches", &RuntimeType::sketches(), exec_state)?;
+    let sketches = args.get_unlabeled_kw_arg(
+        "sketches",
+        &RuntimeType::Union(vec![
+            RuntimeType::sketches(),
```

Comment:
AFAICT, this says the type is either `[[Sketch] | [Face] | [TaggedFace]]` which seems wrong, don't we want `[Sketch | Face | TaggedFace]`?